pkg/server/cors.go

```diff
@@ -0,0 +1,17 @@
+package server
+
+import (
+	"net/http"
+
+	"github.com/spf13/viper"
+)
+
+func (s *server) corsMiddleware(next http.HandlerFunc) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+
+		w.Header().Set("Access-Control-Allow-Origin", viper.GetString("CLIENT_ADDRESS"))
```

Comment:
Doesn't have to happen in this PR, but I realized we can avoid using `viper` as a package, and use `viper.New` and passing an instance instead: https://trussworks.slack.com/archives/CDJ9JAD26/p1579046873213500. We should do this and have it hang off the `sever`. Avoiding the global space is nice.

```
type server struct {
  Config ViperConfig
```

then

```
s.config.GetString("CLIENT_ADDRESS")
```

Just making a note, so we can fix this at some point. Will make a jira ticket.

Reply:
This was quick so I took care of it